CLI/test/ressources/dependenciesNotLoadedInCorrectOrder_2NoZ/output_expected/MainPgm.rdz.tcbl

```diff
@@ -0,0 +1,154 @@
+Simplified Codegen for reference only. DO NOT ATTEMPT TO BUILD, DO NOT DEPLOY !
```

Comment:
This line should be removed.
It was inserted so our users don't build/deploy with this version.
But as we intend to provide a working version of TypeCobol to IBM, the code must compile.

Reply:
Ok, it's done